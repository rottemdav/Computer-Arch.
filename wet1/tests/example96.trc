32 2 20 3 global_history local_tables not_using_share
0xb5b8 T 0xa2e9e14
0x08a260 N 0xdc5fe0
0xb5b8 N 0xa2e9e14
0x08a260 N 0xdc5fe0
0x08a260 N 0xdc5fe0
0x08a260 T 0xdc5fe0
0x08a260 T 0xdc5fe0
0xb5b8 T 0xa2e9e14
0x08a260 N 0xdc5fe0
0xb5b8 N 0xa2e9e14
0xb5b8 N 0xa2e9e14
0xb5b8 T 0xa2e9e14
0x08a260 N 0xdc5fe0
0x08a260 T 0xdc5fe0
0xb5b8 N 0xa2e9e14
0xb5b8 T 0xa2e9e14
0x08a260 T 0xdc5fe0
0xb5b8 N 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0x08a260 T 0xdc5fe0
0xb5b8 N 0xa2e9e14
0xb5b8 N 0xa2e9e14
0x08a260 N 0xdc5fe0
0xb5b8 N 0xa2e9e14
0x08a260 N 0xdc5fe0
0xb5b8 N 0xa2e9e14
0xb5b8 N 0xa2e9e14
0x08a260 N 0xdc5fe0
0xb5b8 T 0xa2e9e14
0xb5b8 N 0xa2e9e14
0xb5b8 T 0xa2e9e14
0x08a260 N 0xdc5fe0
0xb5b8 T 0xa2e9e14
0x08a260 N 0xdc5fe0
0x08a260 N 0xdc5fe0
0xb5b8 T 0xa2e9e14
0x08a260 N 0xdc5fe0
0x08a260 T 0xdc5fe0
0x08a260 N 0xdc5fe0
0xb5b8 T 0xa2e9e14
0x08a260 N 0xdc5fe0
0x08a260 T 0xdc5fe0
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 N 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 N 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 N 0xa2e9e14
0x08a260 N 0xdc5fe0
0xb5b8 T 0xa2e9e14
0x08a260 N 0xdc5fe0
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0x08a260 T 0xdc5fe0
0x08a260 T 0xdc5fe0
0xb5b8 N 0xa2e9e14
0xb5b8 N 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 N 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 N 0xa2e9e14
0xb5b8 N 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 N 0xa2e9e14
0x08a260 N 0xdc5fe0
0x08a260 N 0xdc5fe0
0xb5b8 T 0xa2e9e14
0xb5b8 N 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0x08a260 N 0xdc5fe0
0xb5b8 N 0xa2e9e14
0xb5b8 T 0xa2e9e14
0x08a260 N 0xdc5fe0
0x08a260 N 0xdc5fe0
0xb5b8 N 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0x08a260 T 0xdc5fe0
0x08a260 N 0xdc5fe0
0x08a260 N 0xdc5fe0
0xb5b8 N 0xa2e9e14
0x08a260 T 0xdc5fe0
0x08a260 T 0xdc5fe0
0xb5b8 T 0xa2e9e14
0x08a260 T 0xdc5fe0
0xb5b8 N 0xa2e9e14
0xb5b8 N 0xa2e9e14
0xb5b8 T 0xa2e9e14
0x08a260 N 0xdc5fe0
0x08a260 N 0xdc5fe0
0x08a260 T 0xdc5fe0
0xb5b8 N 0xa2e9e14
0x08a260 N 0xdc5fe0
0x08a260 N 0xdc5fe0
0x08a260 N 0xdc5fe0
0xb5b8 N 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0x08a260 N 0xdc5fe0
0x08a260 N 0xdc5fe0
0x08a260 T 0xdc5fe0
0x08a260 N 0xdc5fe0
0x08a260 T 0xdc5fe0
0xb5b8 T 0xa2e9e14
0x08a260 N 0xdc5fe0
0xb5b8 N 0xa2e9e14
0x08a260 T 0xdc5fe0
0xb5b8 T 0xa2e9e14
0x08a260 T 0xdc5fe0
0xb5b8 N 0xa2e9e14
0x08a260 N 0xdc5fe0
0xb5b8 N 0xa2e9e14
0xb5b8 T 0xa2e9e14
0x08a260 T 0xdc5fe0
0xb5b8 N 0xa2e9e14
0x08a260 N 0xdc5fe0
0xb5b8 N 0xa2e9e14
0xb5b8 T 0xa2e9e14
0x08a260 N 0xdc5fe0
0xb5b8 T 0xa2e9e14