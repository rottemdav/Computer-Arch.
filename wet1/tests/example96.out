0xb5b8 N 0xb5bc
0x8a260 N 0x8a264
0xb5b8 T 0xa2e9e14
0x8a260 T 0xdc5fe0
0x8a260 T 0xdc5fe0
0x8a260 N 0x8a264
0x8a260 T 0xdc5fe0
0xb5b8 T 0xa2e9e14
0x8a260 T 0xdc5fe0
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0x8a260 T 0xdc5fe0
0x8a260 T 0xdc5fe0
0xb5b8 T 0xa2e9e14
0xb5b8 N 0xb5bc
0x8a260 T 0xdc5fe0
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0x8a260 T 0xdc5fe0
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0x8a260 T 0xdc5fe0
0xb5b8 T 0xa2e9e14
0x8a260 N 0x8a264
0xb5b8 T 0xa2e9e14
0xb5b8 N 0xb5bc
0x8a260 N 0x8a264
0xb5b8 N 0xb5bc
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0x8a260 T 0xdc5fe0
0xb5b8 T 0xa2e9e14
0x8a260 T 0xdc5fe0
0x8a260 T 0xdc5fe0
0xb5b8 N 0xb5bc
0x8a260 N 0x8a264
0x8a260 T 0xdc5fe0
0x8a260 N 0x8a264
0xb5b8 T 0xa2e9e14
0x8a260 N 0x8a264
0x8a260 T 0xdc5fe0
0xb5b8 T 0xa2e9e14
0xb5b8 N 0xb5bc
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0x8a260 T 0xdc5fe0
0xb5b8 T 0xa2e9e14
0x8a260 N 0x8a264
0xb5b8 T 0xa2e9e14
0xb5b8 N 0xb5bc
0xb5b8 T 0xa2e9e14
0x8a260 T 0xdc5fe0
0x8a260 T 0xdc5fe0
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 N 0xb5bc
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0x8a260 T 0xdc5fe0
0x8a260 N 0x8a264
0xb5b8 T 0xa2e9e14
0xb5b8 N 0xb5bc
0xb5b8 T 0xa2e9e14
0xb5b8 N 0xb5bc
0xb5b8 N 0xb5bc
0x8a260 T 0xdc5fe0
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0x8a260 N 0x8a264
0x8a260 N 0x8a264
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 N 0xb5bc
0xb5b8 T 0xa2e9e14
0x8a260 T 0xdc5fe0
0x8a260 T 0xdc5fe0
0x8a260 N 0x8a264
0xb5b8 T 0xa2e9e14
0x8a260 N 0x8a264
0x8a260 N 0x8a264
0xb5b8 T 0xa2e9e14
0x8a260 T 0xdc5fe0
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0x8a260 N 0x8a264
0x8a260 N 0x8a264
0x8a260 N 0x8a264
0xb5b8 T 0xa2e9e14
0x8a260 N 0x8a264
0x8a260 T 0xdc5fe0
0x8a260 N 0x8a264
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0xb5b8 N 0xb5bc
0x8a260 T 0xdc5fe0
0x8a260 N 0x8a264
0x8a260 N 0x8a264
0x8a260 N 0x8a264
0x8a260 N 0x8a264
0xb5b8 T 0xa2e9e14
0x8a260 T 0xdc5fe0
0xb5b8 N 0xb5bc
0x8a260 N 0x8a264
0xb5b8 T 0xa2e9e14
0x8a260 N 0x8a264
0xb5b8 T 0xa2e9e14
0x8a260 N 0x8a264
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0x8a260 N 0x8a264
0xb5b8 N 0xb5bc
0x8a260 N 0x8a264
0xb5b8 T 0xa2e9e14
0xb5b8 T 0xa2e9e14
0x8a260 N 0x8a264
0xb5b8 N 0xb5bc
flush_num: 65, br_num: 128, size: 1890b
